assets/src/edit-story/elements/image/index.js

```diff
@@ -42,4 +42,8 @@ export const defaultAttributes = {
   ...MEDIA_DEFAULT_ATTRIBUTES,
 };
 
-export const panels = [...MEDIA_PANELS, PanelTypes.LINK];
+export const panels = [
+  ...MEDIA_PANELS,
+  PanelTypes.IMAGE_ACCESSIBILITY,
+  PanelTypes.LINK,
```

Comment:
According to Figma, the accessibility panel should come after the link panel.

Right now, the accessbility panel seems to be the very first one.

Reply:
Fixed in https://github.com/google/web-stories-wp/pull/652/commits/eef36fbe3357f8343b08bf9f4daa513550c45378